modules/selecting-cluster-capabilities.adoc

```diff
@@ -7,7 +7,14 @@
 = Selecting cluster capabilities
 You can select cluster capabilities by following one of the installation methods that include customizing your cluster, such as "Installing a cluster on AWS with customizations" or "Installing a cluster on GCP with customizations".
 
-During a customized installation, you create an `install-config.yaml` file that contains the configuration parameters for your cluster. You can use the following configuration parameters to select cluster capabilities:
+During a customized installation, you create an `install-config.yaml` file that contains the configuration parameters for your cluster. 
+
+[NOTE]
+====
+If you customize your cluster by enabling, or disabling, specific cluster capabilities, you are responsible for manually maintaining your `install-config.yaml` file. A caveat to manually maintaining your `install-config.yaml` file is that new {product-title} updates might declare new capability handles for existing components, or introduce new components altogether. Users who customize their `install-config.yaml` file should remain mindful of this caveat and consider periodically updating their `install-config.yaml` file as {product-title} is updated.
```

Comment:
I think that you could probably get the point across here in less words. I'll take a stab at it below, but mostly just a suggestion to take another look to see what could be minimized:

```suggestion
If you customize your cluster by enabling or disabling specific cluster capabilities, you are responsible for manually maintaining your `install-config.yaml` file. New {product-title} updates might declare new capability handles for existing components, or introduce new components altogether. Users who customize their `install-config.yaml` file should consider periodically updating their `install-config.yaml` file as {product-title} is updated.
```